config/routes.rb

```diff
@@ -89,6 +89,7 @@
     resource :mass_rollouts, only: [:new, :create, :destroy] do
       collection do
         post :merge
+        get :review_deploy
```

Comment:
little unusual this is a get, but I guess it makes sense since there's no side-effects.